src/Game.h

```diff
@@ -18,29 +25,50 @@ class Game {
 private:
     Game() = default;
 
+    //! Stack containing all states.
     std::stack<std::unique_ptr<State>> states;
 
 public:
+    //! Push state on stack.
     void pushState(std::unique_ptr<State> state);
+    //! Pop state from stack.
     void popState();
```

Comment:
Off topic: shouldn't `popState()` actually return said state?

Reply:
There is no need because after popping states should be destroyed (I think they are because of smart pointer).